app/src/main/java/fr/free/nrw/commons/upload/ThumbnailsAdapter.java

```diff
@@ -85,18 +88,23 @@ public void bind(int position) {
             UploadableFile uploadableFile = uploadableFiles.get(position);
             Uri uri = uploadableFile.getMediaUri();
             background.setImageURI(Uri.fromFile(new File(String.valueOf(uri))));
+            GradientDrawable drawable = new GradientDrawable();
 
             if (position == callback.getCurrentSelectedFilePosition()) {
+                drawable.setShape(GradientDrawable.RECTANGLE);
+                drawable.setStroke(6, ContextCompat.getColor(context, R.color.border_thumbnail_images));
```

Comment:
It might be better to use `R.color.primaryColor` or even `R.color.primaryLightColor` [note 1] here as that would avoid introducing a new colour and would be more in-line with our current colors.

[note 1]: The light variant is nice but we don't seem to be using it anywhere else 🤔 I think we should be using that for the dark theme in place of the light theme's primary color. But that's a topic for a separate issue.